examples/simple/thrift/client.py

```diff
@@ -1,9 +1,11 @@
 import json
```

Comment:
Oh it looks like some of the files don't have license blurbs anymore.

Install the hook by running `sh scripts/install-hooks.sh` in the project root please. @breerly, you too.